lib/ex_base58.ex

```diff
@@ -138,6 +210,24 @@ defmodule ExBase58 do
     end
   end
 
+  @doc """
+  Same as decode_check_version/3, but returns a string value instead of a tuple and raises if the alphabet is wrong
+
+  ## Examples
+
+      iex> ExBase58.decode_check_version!("1GZVwCXwyKVRPTViubJDVKVhVvcaEoX5cN", 0)
+      <<170, 175, 89, 206, 129, 197, 74, 82, 170, 144, 47, 81, 120, 199, 251, 203, 167, 32, 54, 7>>
+
+      iex> ExBase58.decode_check_version!("1GZVwCXwyKVRPTViubJDVKVhVvcaEoX5cN", 0, :not_listed)
+      ** (ArgumentError) Invalid alphabet
+  """
+  @spec decode_check_version!(binary(), non_neg_integer(), alphabet()) :: binary()
+  def decode_check_version!(binary, version, alphabet \\ :bitcoin) do
+    str_alphabet = alphabet_to_string!(alphabet)
+    {:ok, <<^version>> <> decoded} = Impl.decode_check_version(binary, str_alphabet, version)
```

Comment:
I think it would be better to add an explicit exception about a version, but I don't now check it correctly